### PR TITLE
Moved 'Export to Sideloadly' and 'Download more apps' buttons to the menu bar

### PR DESCRIPTION
**Pull request by IsaacMarovitz**
- Moved 'Export to Sideloadly' to File > 'Export to Sideloadly'
- Moved 'Download more apps' to Help > 'Download more apps'

### CODE REVIEW

**Comment by zanderp25** [comment]
I think that the buttons should still show on the main view, because most people probably won't go looking for them in the menus.

File: PlayCover/View/AppsView.swift [hunk 1]
```diff
@@ -88,7 +79,7 @@ struct AppsView: View {
 						}
 					}
 					.padding(.top, 16).padding(.bottom, bottomPadding + 16)
-					.animation(.spring())
+                    .animation(.spring())
```
Comment (zanderp25) [inline PlayCover/View/AppsView.swift:83]: Why was the indentation changed here?

Comment (IsaacMarovitz) [reply PlayCover/View/AppsView.swift:83]: Indentendation wasn't intentionally changed. In Xcode it doesn't show that anything changed.

Comment (zanderp25) [reply PlayCover/View/AppsView.swift:83]: weird

Comment (Depal1) [reply PlayCover/View/AppsView.swift:83]: There were no lint violations, so it should be alright.

**Review by Depal1 — APPROVED** [review]

**Pull request merged**